fabric-testmods/java/net/fabricmc/fabric/events/EntityTickEventMod.java

```diff
@@ -0,0 +1,29 @@
+package java.net.fabricmc.fabric.events;
+
+import net.fabricmc.api.ModInitializer;
+import net.minecraft.entity.EntityType;
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.effect.StatusEffectInstance;
+import net.minecraft.entity.effect.StatusEffects;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.network.chat.Component;
+
+
+public class EntityTickEventMod implements ModInitializer {
```

Comment:
Need to add this testmod's class to initializers

Reply:
This PR is so old that it's got a v0 fabric.mod.json. I'll add it, but it won't do us any good.